backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/spi/impl/AggregateHUTrxListener.java

```diff
@@ -219,6 +222,11 @@ private void updateItemQtyAndSplitIfNeeded(final IHUContext huContext, final Map
 						Quantity.of(qtyToSplit, trx.getQuantity().getUOM()),
 						huContext.getDate());
 				loader.load(request);
+
+				final List<I_M_HU> includedHUs = handlingUnitsDAO.retrieveIncludedHUs(item);
```

Comment:
i have the feeling that loading included HUs here might be a (big) performance penalty.

Reply:
in fact, there will always be just one, so I guess we should be fine.. afaiu, we will only hit this when dealing with an aggregated HU